astroquery/nasa_exoplanet_archive/nasa_exoplanet_archive.py

```diff
@@ -101,29 +101,24 @@ def get_confirmed_planets_table(self, cache=True, show_progress=True,
 
         return self._table
 
-    def query_planet(self, planet_name, table_path=None,
-                     all_columns=False):
+    def query_planet(self, planet_name, **kwargs):
         """
         Get table of exoplanet properties.
 
         Parameters
         ----------
         planet_name : str
             Name of planet
-        table_path : str (optional)
-            Path to a local table file. Default `None` will trigger a
-            download of the table from the internet.
-        all_columns : bool (optional)
-            Return all available columns. The default returns only the
-            columns in the default category at the link above.
+        kwargs : dict (optional)
+            Extra keyword arguments passed to ``get_confirmed_planets_table``.
```

Comment:
I think you can use single backticks here, since it's referring to a method on the same class.  I'm not 100% sure of this though...